listenbrainz/webserver/templates/navbar.html

```diff
@@ -86,7 +84,22 @@
         {%- endif -%}
       </ul>
     {% endif %}
+      <form id="searchForm" class="navbar-form">
+          <input id="searchQuery" type="text" class="form-control input-sm" placeholder="Search Listener">
```

Comment:
Search sounds misleading because there's actually no search. If the user enters a wrong username, they'll be sent to a 404. Something like "Go to Profile" is probably clearer.

Reply:
Sounds reasonable. I'll make the change. Thanks!